engines/experiment/plugin/rpc/manager/rpc_client.go

```diff
@@ -0,0 +1,71 @@
+package manager
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gojek/turing/engines/experiment/manager"
+	"github.com/gojek/turing/engines/experiment/plugin/rpc/shared"
+)
+
+// rpcClient implements ConfigurableExperimentManager interface
+type rpcClient struct {
+	shared.RPCClient
+}
+
+func (c *rpcClient) Configure(cfg json.RawMessage) error {
+	return c.Call("Plugin.Configure", cfg, new(interface{}))
+}
+
+func (c *rpcClient) GetEngineInfo() (resp manager.Engine) {
+	err := c.Call("Plugin.GetEngineInfo", new(interface{}), &resp)
+	if err != nil {
+		// err should be propagated upstream, but it's currently not
+		// possible as GetEngineInfo() on the manager.ExperimentManager
+		// interface doesn't return errors
+		fmt.Printf("plugin errors: %v", err)
+	}
+
+	return resp
+}
+
+func (c *rpcClient) ValidateExperimentConfig(cfg json.RawMessage) error {
+	return c.Call("Plugin.ValidateExperimentConfig", cfg, new(interface{}))
+}
+
+func (c *rpcClient) GetExperimentRunnerConfig(cfg json.RawMessage) (resp json.RawMessage, err error) {
+	err = c.Call("Plugin.GetExperimentRunnerConfig", cfg, &resp)
+	return
+}
+
+func (c *rpcClient) IsCacheEnabled() (resp bool) {
+	_ = c.Call("Plugin.IsCacheEnabled", new(interface{}), &resp)
```

Comment:
Same comment as with `GetEngineInfo` - should we change the signature to capture the error? Perhaps an error from the `rpcClient.IsCacheEnabled` can be discarded, but that decision could be made by the caller.